### DISCUSSION TITLE
Code a real-time web chat

### DISCUSSION THREAD

**Discussion by Pakkia99** [community/community#54117] [Programming Help]
### Body

Hi guys, 
i'm looking for a code for a night pub event,
i'm looking for a web based real time chat where i (admin) can create and delete users, every time that i create a user a popup show a qrcode that i can let scan to every partecipant and it open a web chat where they can see and select all the others partecipants and chat with them individually.
it must be accesible from smartphone with an internet connection
thank for the help

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by jge162** [comment]
You need a developer to create an app like that for you. Do you have experience coding? Otherwise you can try an app store/google play app.

**Reply by Pakkia99** [reply]
HI, it has to be web based, not an app, it has to bo easyly accessible without login
